config/main_cor.yaml

```diff
@@ -0,0 +1,10 @@
+campaign: ORCESTRA
+campaign_id: ORCESTRA
+platform: Sal
+platform_number: 1
+location: Deebles Point
+configs:
+    main: config/main_cor.yaml  # name of this file
+    level0: config/level0_cor.yml
+    level1: config/level1_cor.yaml
+    level2: config/level2.yaml
```

Comment:
This is basically what I have in mind with using different main.yaml. I would just not commit them to this repo but into the post processing one that is specific to the campaign, e.g. ORCHESTRA